packages/ubuntu_desktop_installer/lib/pages/choose_your_look_page.dart

```diff
@@ -12,46 +13,66 @@ class ChooseYourLookPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final lang = AppLocalizations.of(context);
+    final width = MediaQuery.of(context).size.width;
+    final settings = Settings.of(context, listen: false);
     return WizardPage(
       header: Text(lang.chooseYourLookPageHeader),
       actions: <WizardAction>[
         WizardAction.back(context),
         WizardAction.next(context),
       ],
       title: Text(lang.chooseYourLookPageTitle),
-      contentPadding: const EdgeInsets.fromLTRB(20, 50, 20, 150),
       content: Center(
-        child: ListView(
-            shrinkWrap: true,
-            scrollDirection: Axis.horizontal,
-            children: <Widget>[
-              Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: OptionCard(
-                  image: FlavorImage.asset('assets/Theme_thumbnails-Light.png'),
-                  title: Text(lang.chooseYourLookPageLightSetting),
-                  body: Text(lang.chooseYourLookPageLightBodyText),
-                  selected: Theme.of(context).brightness == Brightness.light,
-                  onSelected: () {
-                    final settings = Settings.of(context, listen: false);
-                    settings.applyTheme(Brightness.light);
-                  },
+        child: SingleChildScrollView(
+          child: Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Column(
+                  children: [
+                    SizedBox(
+                      width: width / 3,
+                      child: YaruImageTile(
+                        path: 'assets/Theme_thumbnails-Light.png',
```

Comment:
`FlavorImage.asset` needs to be restored to be able to support flavored installers. I opened a suggestion for YaruImageTile: https://github.com/ubuntu/yaru_widgets.dart/issues/68.